parking_permits/models/order.py

```diff
@@ -465,3 +419,56 @@ def total_payment_price_net(self):
     @property
     def total_payment_price_vat(self):
         return self.total_payment_price * self.vat
+
+
+class SubscriptionStatus(models.TextChoices):
+    CONFIRMED = "CONFIRMED", _("Confirmed")
+    CANCELLED = "CANCELLED", _("Cancelled")
+
+
+class SubscriptionCancelReason(models.TextChoices):
+    RENEWAL_FAILED = "RENEWAL_FAILED", _("Renewal failed")
+    USER_CANCELLED = "USER_CANCELLED", _("User cancelled")
+
+
+class Subscription(SerializableMixin, TimestampedModelMixin, UserStampedModelMixin):
+    order = models.ForeignKey(
+        Order,
+        verbose_name=_("Order"),
+        null=True,
+        blank=True,
+        related_name="subscriptions",
+        on_delete=models.PROTECT,
+    )
+    permit = models.OneToOneField(
+        ParkingPermit,
+        on_delete=models.PROTECT,
+        related_name="subscription",
+        null=True,
+        blank=True,
+    )
+    talpa_subscription_id = models.UUIDField(
+        _("Talpa subscription id"), unique=True, editable=False, null=True, blank=True
```

Comment:
Does this prevent multiple orders to having a same subscription id?

Reply:
It does not prevent that directly, because this is Subscription model itself.
Currently multiple different orders may have the same subscription id.
But I think it still would be better to link subscription to order item instead of order directly. This would probably simplify things even more.

Reply:
Fixed by linking Subscription to OrderItem-model directly.